portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,9 @@
 
 async function getData() {
     const response = await fetch('/data');
-    const data = await response.text();
-    document.getElementById('data-container').innerHTML = data;
+    const comments = await response.json();
+    let html = "<br>";
+    for(let i = 0; i < comments.length; i++)
```

Comment:
If you don't need to know which comment you're currently on (meaning you don't need access to the index), you might want to use
```
for (const comment of comments) {
   html += "<br>" + comment + "<br>";
}
```

That way you don't have to worry about making a mistake with an index out of bounds error.

Reply:
I didn't quite remember how to do for each loops in javascript, thanks for the reminder.